### PR TITLE
Player port

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
This is just a rough port of the project player view from core.

You can view it at: [http://localhost:8000/#/player?user=1&project=1](http://localhost:8000/#/player?user=1&project=1)

File: src/pages/player/player.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+var React = require('react');
+var {parseJSON} = require('../../../node_modules/webmaker-core/src/lib/jsonUtils');
```
Comment (alicoding) [inline src/pages/player/player.jsx:2]: I wonder if there is anyway we can expose these components/lib nicely here without having to access them using relative path which is a bit gross IMO?

Can we maybe optimize this later to have `index.js` file where we can require that will expose each of these or something? what do you think?

Comment (gvn) [inline src/pages/player/player.jsx:2]: That sounds like a good idea, but I think we should do it later in a follow up ticket. Want to make one?

**Comment by alicoding** [comment]
r+

**Pull request merged**